amanuensis/blueprints/inject_notifications.py

```diff
@@ -0,0 +1,40 @@
+import flask
+
+from os import environ
+from amanuensis.config import config
+from amanuensis.auth.auth import current_user
+from amanuensis.errors import AuthError
+from amanuensis.schema import NotificationSchema, NotificationLogSchema
+from amanuensis.resources import notification
+from cdislogging import get_logger
+from pcdcutils.environment import is_env_enabled
+
+blueprint = flask.Blueprint("injectNotification", __name__)
+
+logger = get_logger(__name__)
+
+
+@blueprint.route("/", methods=["GET"])
+def get_all_users():
+    user_notifications = notification.get_notifications()
+    users = []
+    for notif in user_notifications:
+        if notif.user_id not in users:
+            users.append(notif.user_id)
+    
+    return users
```

Comment:
If this is just to retrieve all the users that have at least 1 notification I don't think I can see the meaning. What is this for?